boilerplate/app/theme/styles.ts

```diff
@@ -0,0 +1,19 @@
+import { ViewStyle } from "react-native"
+import { spacing } from "./spacing"
+
+/* Use this file to define styles that are used in multiple places in your app. */
+export const $styles = {
+  row: { flexDirection: "row" } as ViewStyle,
+  flex1: { flex: 1 } as ViewStyle,
+  container: {
+    paddingTop: spacing.lg + spacing.xl,
+    paddingHorizontal: spacing.lg,
+  } as ViewStyle,
```

Comment:
These look to only appear on the demo app screens. We should wrap a comment block around here for removing this block since it'll be unused if/when the user removes the demo code.

You can test the difference with `--remove-demo` (or the Y flag when it asks if you want to remove the demo app) and see that it gets removed or remains accordingly.

```suggestion
  // @demo remove-block-start
  container: {
    paddingTop: spacing.lg + spacing.xl,
    paddingHorizontal: spacing.lg,
  } as ViewStyle,
  // @demo remove-block-end
```

Reply:
@fpena be sure to also add this file to the remove file: `test/vanilla/__snapshots__/ignite-remove-demo.test.ts.snap`